### PR TITLE
ui: new layout of homepage

### PR DESCRIPTION
**Pull request by karolina-siemieniuk-morawska**
+ rewritten related components to Typescript 
+ fix this [TODO](https://github.com/inspirehep/inspirehep/blob/daf8b0f67474176ddafd2969f2b7fb4559f30eec/ui/src/common/containers/SearchScopeSelectContainer.jsx#L16)
* ref: cern-sis/issues-inspire#136

Result:
Screenshot 2022-10-25 at 11 15 01

### CODE REVIEW

File: ui/src/common/components/SearcBoxNamespaceSelect.tsx
Comment (drjova) [inline ui/src/common/components/SearcBoxNamespaceSelect.tsx:1]: `SearcBoxNamespaceSelect` typo in the filename

**Comment by drjova** [comment]
@karolina-siemieniuk-morawska thanks! What about e2e tests? Also could you please add some screenshots in the comments?

**Comment by karolina-siemieniuk-morawska** [comment]
Screen Recording 2022-10-25 at 11 20 47

**Comment by karolina-siemieniuk-morawska** [comment]
News and updates section 
Screenshot 2022-10-25 at 11 24 05

**Comment by karolina-siemieniuk-morawska** [comment]
Mobile view of the new section
Screenshot 2022-10-25 at 14 37 52

**Review by drjova — APPROVED** [review]
LGTM, let's wait for Micha to get a final approval

**Comment by drjova** [comment]
@karolina-siemieniuk-morawska could you please rebase?

**Comment by drjova** [comment]
@karolina-siemieniuk-morawska let's merge it when the tests are finished

**Pull request merged**
